x-pack/plugins/ml/common/util/anomaly_description.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { capitalize } from 'lodash';
+import { getSeverity, type MlAnomaliesTableRecordExtended } from '@kbn/ml-anomaly-utils';
+
+export function getAnomalyDescription(anomaly: MlAnomaliesTableRecordExtended): {
+  anomalyDescription: string;
+  mvDescription: string | undefined;
+} {
+  const source = anomaly.source;
+
+  let anomalyDescription = i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.anomalyInLabel', {
```

Comment:
Nit - you could remove `anomaliesTable` from all these i18n props as they are now being used outside of the table.